### ISSUE TITLE
ConnectivityChanged event always reports true for `IsConnected`

### ISSUE THREAD

**Issue by rmorrin** [jamesmontemagno/Xamarin.Plugins#98]
Hi James, thanks for the project!

I've noticed a small issue with the `ConnectivityChanged` event exposed by the connectivity plugin. The behaviour was observed on Android, but might be present on other platforms too (I was unfortunately unable to trigger the event from the WP emulator).

The problem can be reproduced by adding some delegate to the `ConnectivityChanged` event, then inspecting `IsConnected` on the provided event args after connection is lost (both aeroplane mode and disabling wifi give the same result).

```
CrossConnectivity.Current.ConnectivityChanged += (sender, args) =>
{
    System.Diagnostics.Debug.WriteLine($"Event args: {args.IsConnected}"); // true
    System.Diagnostics.Debug.WriteLine($"Plugin: {CrossConnectivity.Current.IsConnected}"); // false
};
```

Looks like `IsConnected` is always reporting true, I've checked against the same property provided by the plugin: `CrossConnectivity.Current.IsConnected` which is false as expected.

Cheers!

EDIT: I should add that is under Android 5.0, both emulator and device.

**Comment by jamesmontemagno** [comment]
Looking into this. Apparently something broke in 5.0, testing a new alpha package.

**Comment by jamesmontemagno** [comment]
Fixed in 1.0.3
